hubstorage/client.py

```diff
@@ -88,10 +88,12 @@ def request(self, is_idempotent=False, **kwargs):
         def invoke_request():
             r = self.session.request(**kwargs)
 
-            if not r.ok:
+            try:
+                r.raise_for_status()
+                return r
+            except HTTPError:
                 logger.debug('%s: %s', r, r.content)
-            r.raise_for_status()
-            return r
+                raise e
```

Comment:
@qrilka `e` is undefined, you simply want

```
raise
```

here. I didn't notice that yesterday, sorry :)